test/sets.jl

```diff
@@ -46,7 +50,7 @@ end
         @test isdisjoint(setdiff(a, b), b)
         @test !isdisjoint(a, a)
 
-        intersections = find_intersections(a, b)
+        intersections = find_intersections(Array(a), Array(b))
```

Comment:
Does `collect` not work here?

Reply:
yes, though `Array` is faster since it avoids the copy.

Reply:
For this case you could just dispatch on `IntervalSet` to and avoid the copy that way